src/Cli/dotnet/commands/dotnet-tool/install/LocalizableStrings.resx

```diff
@@ -232,4 +232,10 @@ If you would like to create a manifest, use `dotnet new tool-manifest`, usually
   <data name="PrereleaseAndVersionAreNotSupportedAtTheSameTime" xml:space="preserve">
     <value>The --prerelease and --version options are not supported in the same command</value>
   </data>
+  <data name="CreateManifestIfNeededOptionDescription" xml:space="preserve">
+    <value>Create tool manifest when installing tools if manifest not exist.</value>
```

Comment:
@baronfel Any suggestions for this option description?  Should we describe how we decide where to put the manifest file?

Reply:
How about "Create a tool manifest if one isn't found during tool installation. For information on how manifests are located, see [HELP URL]".

We should document the search algorithm on https://learn.microsoft.com/en-us/dotnet/core/tools/global-tools#install-a-local-tool, though. Potentially on the [CannotFindAManifestFile](https://github.com/dotnet/sdk/pull/31231/files/6177c345e89b2357ab290c0d541fe55e209733e0#diff-4288e2d70efc0eba5c7ffec9c84313f69b4111f7611d00345c58442ef96b690aR120-R123) error as well.

Reply:
How about https://aka.ms/dotnet-local-tool-create-manifest-if-needed?

Reply:
An aka.ms link would be perfect - folks have been 'namespacing' them using url path segments, though - maybe something like `aka.ms/dotnet/tools/create-manifest-if-needed`?